test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -94,8 +100,7 @@ void Interpreter::operator()(VariableDeclaration const& _declaration)
 		YulString varName = _declaration.variables.at(i).name;
 		solAssert(!m_variables.count(varName), "");
 		m_variables[varName] = values.at(i);
-		solAssert(!m_scopes.back().count(varName), "");
```

Comment:
We cannot have these checks anymore because the scope is not cleaned up (on purpose) when we leave it. So if we re-execute the same scope, the variables will already be there.